packages/ui/src/components/Form/expression/ExpressionEditor.tsx

```diff
@@ -57,6 +57,7 @@ export const ExpressionEditor: FunctionComponent<ExpressionEditorProps> = ({
           isOpen={isOpen}
           selected={selectedLanguage.model.name}
           onSelect={onSelect}
+          onOpenChange={(isOpen: boolean) => setIsOpen(isOpen)}
```

Comment:
I think we can pass the function directly to the property 

```suggestion
          onOpenChange={setIsOpen}
```

Reply:
Thanks @lordrip for the hint - updated